src/ngx-gallery-scroll-overview/ngx-gallery-scroll-overview.component.html

```diff
@@ -0,0 +1,8 @@
+<div id="previewContainer" class="drag-boundary image-box"
+     [ngClass]="{ 'top-b': !edge?.top, 'bottom-b': !edge?.bottom, 'left-b': !edge?.left, 'right-b': !edge?.right }"
+     #myBounds>
+    <div id="zoomContainer" ngDraggable class="drag-block image-detail" (started)="onStart($event)"
+         (stopped)="onStop($event)"
+         (edge)="checkEdge($event)" [bounds]="myBounds" [inBounds]="inBounds">
```

Comment:
inBounds could be set to true here because we do not want to change this. So `inBounds="true"`